[2024-01-31T12:16:24.996+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_analysis_dag.get_and_store_gpt_advice manual__2024-01-31T12:16:10.065968+00:00 [queued]>
[2024-01-31T12:16:25.000+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_analysis_dag.get_and_store_gpt_advice manual__2024-01-31T12:16:10.065968+00:00 [queued]>
[2024-01-31T12:16:25.000+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-31T12:16:25.011+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_and_store_gpt_advice> on 2024-01-31 12:16:10.065968+00:00
[2024-01-31T12:16:25.014+0000] {standard_task_runner.py:60} INFO - Started process 3201 to run task
[2024-01-31T12:16:25.016+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'stock_analysis_dag', 'get_and_store_gpt_advice', 'manual__2024-01-31T12:16:10.065968+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/stock_analysis_dag.py', '--cfg-path', '/tmp/tmpfh3hvzsh']
[2024-01-31T12:16:25.017+0000] {standard_task_runner.py:88} INFO - Job 45: Subtask get_and_store_gpt_advice
[2024-01-31T12:16:25.038+0000] {task_command.py:423} INFO - Running <TaskInstance: stock_analysis_dag.get_and_store_gpt_advice manual__2024-01-31T12:16:10.065968+00:00 [running]> on host de-airflow-vm-instance-1.europe-west10-a.c.dataengineeringfinal-411911.internal
[2024-01-31T12:16:25.080+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='onatpostac@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='stock_analysis_dag' AIRFLOW_CTX_TASK_ID='get_and_store_gpt_advice' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T12:16:10.065968+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T12:16:10.065968+00:00'
[2024-01-31T12:16:26.445+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-31T12:16:26.446+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-31T12:16:26.649+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T12:16:26.655+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=stock_analysis_dag, task_id=get_and_store_gpt_advice, execution_date=20240131T121610, start_date=20240131T121624, end_date=20240131T121626
[2024-01-31T12:16:26.671+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T12:16:26.679+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
